manual/wixtoolset/tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,13 @@
-﻿$packageName = 'wixtoolset'
-$installerType = 'EXE'
-$32BitUrl = 'http://wixtoolset.org/downloads/v3.10.3.3007/wix310.exe'
-$silentArgs = '/q'
-$validExitCodes = @(0)
+﻿$packageParameters = @{
+  PackageName = 'wixtoolset'
+  SilentArgs = '/q'
+  Url = 'http://wixtoolset.org/downloads/v3.11.0.1507/wix311.exe'
+  Url64bit = ''
+  ValidExitCodes = @('0')
+  Checksum = '407e00a2c8787c347e6963091d066b0e2530866a161a59a88c56817cf88e9367'
+  ChecksumType = 'Sha256'
+  Checksum64 = ''
+  ChecksumType64 = ''
```

Comment:
please remover the 64-bit variables, they are not needed and therefore should not be used.